### ISSUE TITLE
Should the wit crate be relocated to wasm-pkg-tools?

### ISSUE THREAD

**Issue by ricochet** [bytecodealliance/cargo-component#307]
Now that there is a hosted project for [Wasm packaging tools](https://github.com/bytecodealliance/wasm-pkg-tools), should the [wit crate](https://github.com/bytecodealliance/cargo-component/tree/main/crates/wit) in this project be moved there? I imagine most of the maintenance and iteration will be closely tied to the crates in [wasm-pkg-tools](https://github.com/bytecodealliance/wasm-pkg-tools).

**Comment by fibonacci1729** [comment]
That is the plan! AFAICT it's not a simple lift and shift however and will require some refactoring to eliminate a dependency on `cargo-component-core`. I haven't dug too deeply into it yet.

**Comment by calvinrp** [comment]
Yep, an inspired rewrite should land in `wasm-pkg-tools` repo during the month of June.
